docker-compose.dev.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.8'
+services:
+  pic-sure-auth-service:
+    build:
+        context: .
+        dockerfile: ./pic-sure-auth-services/dev.Dockerfile
+    environment:
+      JAVA_OPTS: "-Xmx1024m -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
+      # Note: To use this you will need to set ENVIRONMENT variables for the application.properties
+    ports:
+      - "8090:8090" # Application port
+      - "5005:5005" # Debug port
+    depends_on:
+      auth-flyway:
+        condition: service_started
+
+  picsure-db:
```

Comment:
Did you mean to give this the same container name as the db I use by default in the all in one?

Reply:
No, this was intended for local development. Not development on an all in one.